packages/calypso-build/babel.config.js

```diff
@@ -0,0 +1,57 @@
+const config = {
+	presets: [
+		[
+			'@babel/env',
+			{
+				useBuiltIns: 'entry',
+				corejs: 2,
```

Comment:
Should this be parametrized? 

It's set `false` in extensions babel config: https://github.com/Automattic/jetpack/blob/7526a5ea2c58604eb1305d65736984ec97493400/babel.config.extensions.js#L22

Reply:
Hmm, actually that was for `@babel/transform-runtime` not for `@babel/env`. I'm not super familiar with Babel so on uncertain ground here... :-)

Reply:
Yep, different Babel plugin. We still have it in this file as well BTW: https://github.com/Automattic/wp-calypso/pull/31464/files#diff-0e759b21864de1336af9c450cff5ad20R22

And I also don't really know what it does. Will look it up :slightly_smiling_face:

Reply:
@jsnajdr can probably help here as he was working on changes to this setting in Calypso and Jetpack.

Reply:
The `corejs` param should be set to `2` here. Since last week, there is also version `3` and the latest Babel will issue a big warning if `corejs` is not set explicitly. Version 2 used to be a reliable default. It's still is the default, but not guaranteed in future.

That's for the `@babel/env` preset. It will cherry-pick only the CoreJS polyfills that are needed according to the browserslist. Without the preset, the bundle would include the complete CoreJS library.

There's also `corejs` option for `@babel/transform-runtime` plugin. And that one we want set to `false`. What it does is that it finds code like:
```js
const cache = new Map();
Promise.resolve( 1 );
```
and transforms it to:
```js
import _Map from '@babel/runtime/core-js/map';
import _Promise from '@babel/runtime/core-js/promise';

const cache = new _Map();
_Promise.resolve( 1 );
```
I.e., the global references are transformed into imported references. But we don't want that at all. Our polyfills are added to the global namespace. And we have no reason to complicate things and avoid that "pollution".